Source/MQTTnet/Client/Options/MqttClientTlsOptions.cs

```diff
@@ -26,7 +26,11 @@ public class MqttClientTlsOptions
         public List<SslApplicationProtocol> ApplicationProtocols { get; set; }
 #endif
 
+#if NETCOREAPP3_0 || NET5_0
+        public SslProtocols SslProtocol { get; set; } = SslProtocols.Tls13;
```

Comment:
Is 1.3 backward compatible with 1.2? I have doubts if users can still connect to their servers after this update. Especially when they leave the initial values untouched (so 1.2).